wix/Austin.wxs

```diff
@@ -78,6 +78,7 @@
       ConfigurableDirectory='INSTALLDIR'>
       <ComponentRef Id='MainExecutable' />
     </Feature>
-
+    <MajorUpgrade
+      DowngradeErrorMessage="A later version of Austin is already installed. Setup will now exit."/>
```

Comment:
Is this all that's required to fix #88? Based on the error message I would have thought that `chocolateyInstall.ps1` also needed to be fixed?

Reply:
The chocolatey installer just fires MSIexec, the error message is actually coming from the MSI. 

WiX needs instructions to remove old versions, and this is the easiest way to do it. I haven't been able to test it though, as it would require building a new MSI. If you can build a new version I can test doing the upgrade.